web/packages/core/src/shadow-template.ts

```diff
@@ -782,6 +782,26 @@ const videoModalClose = createElement("span", undefined, "close-modal");
 videoModalClose.textContent = "\u00D7";
 const videoHolder = createElement("div", "video-holder");
 
+// Hardware acceleration modal elements
+const hardwareModal = createElement(
+    "div",
+    "hardware-acceleration-modal",
+    "modal hidden",
+);
+const hardwareModalArea = createElement("div", undefined, "modal-area");
+const hardwareModalClose = createElement("span", undefined, "close-modal");
+hardwareModalClose.textContent = "\u00D7";
+const hardwareModalSpanOne = createElement("span");
+hardwareModalSpanOne.textContent =
+    "It looks like hardware acceleration is not enabled. While Ruffle may work, it could be unreasonably slow. You can enable hardware acceleration by ";
```

Comment:
It'd be nice to localize this, but the way it's constructed won't be trivial to split up. Not sure what to do there 🤔

Reply:
It can be: "It looks like hardware acceleration is not enabled. While Ruffle may work, it could be unreasonably slow. You can enable hardware acceleration by following this link: [link]". It would be a single string of text to translate then (since the link doesn't need translating), though it's a little uglier. Thoughts?

Reply:
Updated, see screencast
[Screencast from 2023-10-02 10-22-41.webm](https://github.com/ruffle-rs/ruffle/assets/29206584/e1a72dd2-46a8-4949-8e78-52694d9697b8)